shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -1272,65 +1284,220 @@ - (void)updateViewportPadding {
 
 #pragma mark - Keyboard events
 
+typedef NS_ENUM(NSInteger, FlutterKeyboardMode) {
```

Comment:
Put this in [FlutterViewController_Internal.h](https://github.com/flutter/engine/blob/main/shell/platform/darwin/ios/framework/Source/FlutterViewController_Internal.h) so it can be used directly by the tests instead of `NSInteger`.
Alternatively you could forward declare `typedef NS_ENUM(NSInteger, FlutterKeyboardMode);` in the test, but `FlutterViewController_Internal` is specifically a non-public header to be used by the rest of the code, including tests, but not included in the exported API headers.